lib/transforms/json-to-html-data.js

```diff
@@ -325,13 +325,10 @@ function getObjectType (data, links) {
   var value;
   if (data.type !== 'people' && data.attributes.name) {
     value = data.attributes.name.map((el, i, arr) => {
-      var val = el.value;
```

Comment:
All of the values in the details block at the bottom, there's a helper that does it. I didn't apply it to the values at the top of the page